### PR TITLE
Add 25 arcsec to margin for fid spoiling per #251

### LINKED ISSUE #251: Fid in acq box check should have margin for fid position uncertainty
Probably 25 arcsec is reasonable.  This accounts for the actual position uncertainty < 15 arcsec plus a couple of pixels.  (E.g. actual PEA readout box can be a pixel larger than expected due to 2x2 binning, and the fid has a broad PSF).

image

https://github.com/sot/starcheck/blob/5879961efec6378b5900f34714c04a193cba22e8/starcheck/src/lib/Ska/Starcheck/Obsid.pm#L1305

### PR DESCRIPTION
**Pull request by taldcroft**
Closes #251

### CODE REVIEW

File: starcheck/src/lib/Ska/Starcheck/Obsid.pm [hunk 1]
```diff
@@ -1438,9 +1438,16 @@ sub check_star_catalog {
 
         # Check for situation that occurred for obsid 14577 with a fid light
         # inside the search box (PR #50).
+
         if ($type =~ /BOT|ACQ/){
+            # Margin for fid spoiling the acquisition star is the search box halfwidth
+            # plus the uncertainty in fid position.  See starcheck #251 for justification
+            # of the 25 arcsec value here.
+            $fid_spoil_margin = $halfw + 25.0
```
Comment (jeanconn) [inline starcheck/src/lib/Ska/Starcheck/Obsid.pm]: Needs an initialization with a 'my'.  I can patch.

Comment (jeanconn) [reply starcheck/src/lib/Ska/Starcheck/Obsid.pm]: Oh and semicolon. :smile:

**Comment by taldcroft** [comment]
Yes, I cannot perl any more.  :smile:

**Comment by jeanconn** [comment]
I don't have a good way to quickly and explicitly check that this works (with a fid light *almost* in the acq box) but can confirm the test still works on 14577 in JUL2913A.
```
 >> WARNING: [ 7] Fid light in search box
```

**Pull request merged**